src/styles/globals.css

```diff
@@ -135,6 +138,7 @@ body {
     gap: 15px;
     padding: 5px;
     margin-bottom: 30px;
+    margin-top: 30px;
```

Comment:
Wenn man einen Bereich hat, in dem man meherere Bereiche untereinander (oder auch hintereinander) weg listet, ist es grundsätzlich eine gute Idee, wenn man mit Margins arbeitet, sich für margin-bottom oder margin-top zu entscheiden, und damit konsequent zu gehen. Der GoTo Weg, den die meisten benutzen, ist margin-bottom.